src/utils.js

```diff
@@ -39,7 +39,7 @@ function invokeArrayArg(arg, fn, context) {
  * walk objects and arrays
  * @param {Object} obj
  * @param {Function} iterator
- * @param {Object} context
+ * @param {Object=null} [context]
```

Comment:
this is not the correct format for a default value with jsdoc should be 
`* @param {Object} [context=null]`